webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/ZeebeESConstants.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.operate.zeebe;
+package io.camunda.webapps.schema.descriptors;
```

Comment:
❓ Why we need this?

Reply:
The importers uses these enums to write into the import position index, and so for the exporter to read from the index to know which record reader is done the enums are easiest to use without introducing magic strings because for example.

I think I will take the approach where I avoid moving it and create my own import value type -> index name mapping